docs/api/rtk-query/created-api/hooks.md

```diff
@@ -78,87 +231,281 @@ type UseQueryResult<T> = {
+type UseLazyQueryTrigger = (arg: any) => void
+
+type UseQueryStateResult<T> = {
+  // Base query state
+  originalArgs?: unknown // Arguments passed to the query
+  data?: T // Returned result if present
+  error?: unknown // Error result if present
+  requestId?: string // A string generated by RTK Query
+  endpointName?: string // The name of the given endpoint for the query
+  startedTimeStamp?: number // Timestamp for when the query was initiated
+  fulfilledTimeStamp?: number // Timestamp for when the query was completed
+
+  isUninitialized: false // Query has not started yet.
+  isLoading: false // Query is currently loading for the first time. No data yet.
+  isFetching: false // Query is currently fetching, but might have data from an earlier request.
+  isSuccess: false // Query has data from a successful load.
+  isError: false // Query is currently in an "error" state.
+}
+
+type UseLazyQueryLastPromiseInfo = {
+  lastArg: any
+}
+```
+
+- **Parameters**
+
+  - `options`: A set of options that control the fetching behavior and returned result value of the hook. Options affecting fetching behavior will only have an effect after the lazy query has been triggered at least once.
+
+- **Returns**: A tuple containing:
+  - `trigger`: A function that fetches the corresponding data for the endpoint when called
+  - `result`: A query result object containing the current loading state, the actual data or error returned from the API call and metadata about the request. Can be customized with `selectFromResult`
+  - `lastPromiseInfo`: An object containing the last argument used to call the trigger function
+
+#### Description
+
+[summary](docblock://query/react/buildHooks.ts?token=UseLazyQuery)
 
 ## `useLazyQuerySubscription`
 
-TBD
+#### Signature
+
+```ts
+type UseLazyQuerySubscription = (
+  options?: UseLazyQuerySubscriptionOptions
+) => [UseLazyQuerySubscriptionTrigger, LastArg]
+
+type UseLazyQuerySubscriptionOptions = {
+  pollingInterval?: number
+  refetchOnReconnect?: boolean
+  refetchOnFocus?: boolean
+}
+
+type UseLazyQuerySubscriptionTrigger = (arg: any) => void
+```
+
+- **Parameters**
+
+  - `options`: A set of options that control the fetching behavior of the hook. The options will only have an effect after the lazy query has been triggered at least once.
+
+- **Returns**: A tuple containing:
+  - `trigger`: A function that fetches the corresponding data for the endpoint when called
+  - `lastArg`: The last argument used to call the trigger function
+
+#### Description
+
+[summary](docblock://query/react/buildHooks.ts?token=UseLazyQuerySubscription)
 
 ## `usePrefetch`
 
-TBD
+#### Signature
+
+```ts
+type UsePrefetch = (
+  endpointName: string,
+  options?: UsePrefetchOptions
+) => PrefetchCallback
+
+type UsePrefetchOptions =
+  | {
+      // If specified, only runs the query if the difference between `new Date()` and the last
+      // `fulfilledTimeStamp` is greater than the given value
+      ifOlderThan?: false | number
```

Comment:
Should we note that this value is in seconds here?

Reply:
Yep, good point. Done now